### ISSUE TITLE
Data not available other month

### ISSUE THREAD

**Issue by balran** [tattali/CalendarBundle#12]
Hey, 

When I create a new event at the end of the month my event is not available on next month.

Ex: If I create an event on the 29 septembre to 4 October, I saw it in the September, if I swap to October I didn't see the event on the 1,2,3,4 October. 

Any idea ? 

Regards,

**Comment by tattali** [comment]
Hi, 
I fix this issue by remove the where clause on the booking query
```php
$bookings = $this->bookingRepository
    ->createQueryBuilder('booking')
    // ->where('booking.beginAt BETWEEN :start and :end')
    // ->setParameter('start', $start->format('Y-m-d H:i:s'))
    // ->setParameter('end', $end->format('Y-m-d H:i:s'))
    ->getQuery()
    ->getResult()
;
```
this method is not optimised at all

**Comment by balran** [comment]
Hey,

Thanks for your response!

Very good job about your bundle !

Regard,

Obtenez Outlook pour iOS<https://aka.ms/o0ukef>
________________________________
De : tattali <notifications@github.com>
Envoyé : Friday, September 27, 2019 5:49:00 PM
À : tattali/CalendarBundle <CalendarBundle@noreply.github.com>
Cc : Mysad <mathieudevinez@hotmail.fr>; Author <author@noreply.github.com>
Objet : Re: [tattali/CalendarBundle] Data not available other month (#12)


Hi,
I fix this issue by remove the where clause on the booking query

$bookings = $this->bookingRepository
    ->createQueryBuilder('booking')
    // ->where('booking.beginAt BETWEEN :start and :end')
    // ->setParameter('start', $start->format('Y-m-d H:i:s'))
    // ->setParameter('end', $end->format('Y-m-d H:i:s'))
    ->getQuery()
    ->getResult()
;

this method is not optimised at all

**Comment by tattali** [comment]
Merci ;)
